internal/services/v1/relationships.go

```diff
@@ -81,13 +97,17 @@ func NewPermissionsServer(
 	config PermissionsServerConfig,
 ) v1.PermissionsServiceServer {
 	configWithDefaults := PermissionsServerConfig{
-		MaxPreconditionsCount:      defaultIfZero(config.MaxPreconditionsCount, 1000),
-		MaxUpdatesPerWrite:         defaultIfZero(config.MaxUpdatesPerWrite, 1000),
-		MaximumAPIDepth:            defaultIfZero(config.MaximumAPIDepth, 50),
-		StreamingAPITimeout:        defaultIfZero(config.StreamingAPITimeout, 30*time.Second),
-		MaxCaveatContextSize:       defaultIfZero(config.MaxCaveatContextSize, 4096),
-		MaxRelationshipContextSize: defaultIfZero(config.MaxRelationshipContextSize, 25_000),
-		MaxDatastoreReadPageSize:   defaultIfZero(config.MaxDatastoreReadPageSize, 1_000),
+		MaxPreconditionsCount:            defaultIfZero(config.MaxPreconditionsCount, 1000),
+		MaxUpdatesPerWrite:               defaultIfZero(config.MaxUpdatesPerWrite, 1000),
+		MaximumAPIDepth:                  defaultIfZero(config.MaximumAPIDepth, 50),
+		StreamingAPITimeout:              defaultIfZero(config.StreamingAPITimeout, 30*time.Second),
+		MaxCaveatContextSize:             defaultIfZero(config.MaxCaveatContextSize, 4096),
+		MaxRelationshipContextSize:       defaultIfZero(config.MaxRelationshipContextSize, 25_000),
+		MaxDatastoreReadPageSize:         defaultIfZero(config.MaxDatastoreReadPageSize, 1_000),
+		MaxReadRelationshipsLimit:        defaultIfZero(config.MaxReadRelationshipsLimit, 1_000),
+		MaxDeleteRelationshipsLimit:      defaultIfZero(config.MaxDeleteRelationshipsLimit, 1_000),
+		MaxLookupResourcesLimit:          defaultIfZero(config.MaxLookupResourcesLimit, 1_000),
+		MaxBatchExportRelationshipsLimit: defaultIfZero(config.MaxBatchExportRelationshipsLimit, 100_000),
```

Comment:
didn't you use 10_000 as the new default in `internal/services/v1/experimental.go`?

Reply:
And I set the parameter to that value; this was the fallback default